make TODOs visible

flash62au opened the pull request:
List of TODOs is now at the end of the about/roadmap page.

flash62au committed:
Update conf.py

flash62au committed:
Update conf.py

flash62au committed:
Update docs.yml

flash62au committed:
Update docs.yml

flash62au committed:
sphinx branch

flash62au committed:
todo

flash62au committed:
yml

flash62au committed:
Update docs-flash62au-restruct.yml

flash62au committed:
Update docs-flash62au-restruct.yml

flash62au committed:
spaces

flash62au committed:
Update docs-flash62au-restruct.yml

flash62au committed:
Update docs-flash62au-restruct.yml

flash62au committed:
Update docs-flash62au-restruct.yml

flash62au committed:
Update docs-flash62au-restruct.yml

flash62au committed:
Update conf.py

flash62au committed:
Update conf.py

flash62au commented:
the additional manual workflow is also added here.  but is not usable from this branch.  It needs to be in the sphinx branch.

FrightRisk reviewed (approved):
Another good cleanup and improvement. Especially since this is in a dev branch, we can merge it to flash62au-restruct

flash62au committed:
additional expansions

expansions for
- Engine Driver
- wiThrottle
- wiThrottle Protocol

more reference to DCC++EX changed to expansions

flash62au committed:
fixes for the previous commit

flash62au committed:
lots of changes of the -EX suffix to a EX- prefix
